kie-server-parent/kie-server-services/kie-server-services-openshift/src/main/java/org/kie/server/services/api/KieServerReadinessProbe.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2018 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+package org.kie.server.services.api;
```

Comment:
can we update package name to reflect the artifact id and to avoid package split?

so in this case it would be `org.kie.server.services.openshift.api` this applies to all classes in this new module

Reply:
Refactored all code into org.kie.server.services.openshift subpackages.